lib/src/test/java/com/auth0/jwt/JWTCreatorTest.java

```diff
@@ -157,7 +158,41 @@ public void shouldNotOverwriteKeyIdIfAddedFromRSAAlgorithms() throws Exception {
         String headerJson = new String(Base64.decodeBase64(parts[0]), StandardCharsets.UTF_8);
         assertThat(headerJson, JsonMatcher.hasEntry("kid", "my-key-id"));
     }
-
+    
+    @Test
+    public void shouldAddKeyIdIfAvailableFromECDSAKAlgorithms() throws Exception {
+        ECPrivateKey privateKey = (ECPrivateKey) PemUtils.readPrivateKeyFromFile(PRIVATE_KEY_FILE_EC_256K
```

Comment:
weird formatting?

Reply:
Pushed a commit to fix running the formatter.